tests/coreHTTP/run-cbmc-proofs.patch

```diff
@@ -0,0 +1,39 @@
+From 20cd5c5902658332a3cc11e1b1c9df2b36eddc46 Mon Sep 17 00:00:00 2001
+From: "Mark R. Tuttle" <mrtuttle@amazon.com>
+Date: Sun, 17 Oct 2021 09:32:18 -0400
+Subject: [PATCH] patch
+
+---
+ template-for-repository/proofs/run-cbmc-proofs.py | 9 +++++++--
+ 1 file changed, 7 insertions(+), 2 deletions(-)
+
+diff --git a/template-for-repository/proofs/run-cbmc-proofs.py b/template-for-repository/proofs/run-cbmc-proofs.py
+index 3ef9c60..24d7c06 100755
+--- a/template-for-repository/proofs/run-cbmc-proofs.py
++++ b/template-for-repository/proofs/run-cbmc-proofs.py
+@@ -196,6 +196,12 @@ def run_build(litani, jobs):
+         sys.exit(1)
+ 
+ 
++def strip_make_output(stdout):
++    lines = [line for line in stdout.splitlines()
++             if not line.startswith('make')]
++    assert(len(lines) == 1)
++    return lines[0].strip()
++
+ def get_litani_path(proof_root):
+     cmd = [
+         "make",
+@@ -208,8 +214,7 @@ def get_litani_path(proof_root):
+     if proc.returncode:
+         logging.critical("Could not determine path to litani")
+         sys.exit(1)
+-    return proc.stdout.strip()
+-
++    return strip_make_output(proc.stdout)
+ 
+ def get_litani_capabilities(litani_path):
+     cmd = [litani_path, "print-capabilities"]
+-- 
+2.30.1 (Apple Git-130)
+
```

Comment:
We should open a GitHub issue at least to request this update on Litani. Why would we need an specific version of this script just for these regression tests?